pages/about_data_page.py

```diff
@@ -1,12 +1,242 @@
+                The 'comparing scenarios' page compares lifetime costs of gas boilers and air source heat pumps under the above pre-set scenarios.
+
+                Below you can find more information about the methodology and data sources used in the lifetime costs calculator.
+                """
+    )
+    
+    st.markdown("## Methodology")
+
+    st.markdown(f"""Methodology for calculating lifetime costs of heating systems. 
+                Parameters highlighted in <span style='color:{nesta_blue};'>blue</span> are changeable inputs by the user in the app.""", unsafe_allow_html=True)
+
+    with st.expander("Total lifetime costs"):
+        # Lifetime cost calculations
+        st.markdown(f"""
+                    **Total lifetime cost**: This is the total cost of owning a heating system over its entire lifetime.
+                    
+                    Total lifetime cost [£] = Upfront costs [£] + Maintenance costs over lifetime [£] + Running costs over lifetime [£]
+
+                    **Annualised lifetime cost**: This is the lifetime cost averaged over the heating system's lifetime.
+
+                    Annualised lifetime cost [£/year] = Total lifetime cost [£] / <span style='color:{nesta_blue};'>Lifespan</span> [years]
+
+                    **Time series breakdown of total cost over the heating system's lifetime**: This is the total cost of having that heating system in each year of its lifetime. This cost will be different for each year of its lifetime.
+
+                    For each year of the heating system's lifetime:
+
+                    Total cost of owning the heating system [£/year] = Upfront cost [£/year] + Maintenance cost [£/year] + Running cost [£/year]
+                    """)
+        
+    with st.expander("Upfront costs"):
+        st.markdown(f"""
+            **Adjusting installation cost based on year of purchase**:
+            
+            For each year:
+                    
+            Cost reduction = (1 - <span style='color:{nesta_blue};'>annual cost reduction rate</span>) * Cost reduction in previous year
+                    
+            where span style='color:{nesta_blue};'>annual cost reduction rate</span> is how much we expect cost to reduce annualy e.g. 0.01 if 1% reduction expected
+                    
+            Adjusted installation cost [£] = Installation cost x cost reduction 
+            
+            **If purchasing with loan, calculate total loan repayment value**:
+            
+            Loan ammount =  Adjusted installation cost - Subsidy available
+                    
+            Annual loan payment = (Loan ammount x <span style='color:{nesta_blue};'>interest rate</span>) / (1 - ((1 + <span style='color:{nesta_blue};'>interest rate</span>) ** -<span style='color:{nesta_blue};'>lifespan/span>))
+            
+            Loan repayment value = Annual loan payment x <span style='color:{nesta_blue};'>lifespan/span>
```

Comment:
```suggestion
            Loan repayment value [£] = Annual loan payment [£/year] x <span style='color:{nesta_blue};'>Lifespan</span> [years]
```

We should agree on a convention but just suggesting to capitalise each parameter for now, for readability